README.md

```diff
@@ -14,15 +14,27 @@ should contain details about expected upload, which include additional metadata,
 about content type and size, and callback URL which will be used to notify user. Here is an example of the request:
 ```
 {
-	"callbackUrl": "http://myservice.com/callback?fileId=123"
+	"callbackUrl": "http://myservice.com/callback?fileId=123",
+	"minimumFileSize" : 0,
+	"maximumFileSize" : 1024,
+	"expectedMimeType": "application/xml"
```

Comment:
is there only one possible mime type per upload?

Reply:
Well - :) That's the limitation of AWS :)

Reply:
what is the size unit? Bytes? Can you specify with prefix e.g. 100M, or 10KB?

Reply:
Integer, bytes

Reply:
would it make sense to document it in the readme?